LukeMason33/CharityLocal
# Feature/testing

### Is this a fix or a feature?
- This is a feature that runs the end-to-end testing using Cypress.

### What is the change/fix?
- The change is the installation of Cypress, and the implementation of user-story guided tests.

### Where should the reviewer start?
- The reviewer should start in `cypress/integration/charityLocal.js`. Here is where all the tests live.

### How should this be tested?
- This should be tested by running `npm i` in the console to ensure you install Cypress, and then run `npx cypress open` and ensure all tests are passing.
